### ISSUE TITLE
not able to convert .pt into ONNX

### ISSUE THREAD

**Issue by apanand14** [ultralytics/yolov5#6823]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello,

Currently, I'm facing this issue while exporting the model. Earlier, I was able to convert to torchscript and ONNX. But now on, it doesn't convert to the ONNX and doens't produce any error as well. Please help me with this. Thank you.

**(yolo) C:\Developer\yolo\yolov5>python export.py --weights Z:\yolov5\train\exp6_640_100e\weights\best.pt
export: data=data\coco128.yaml, weights=['Z:\\yolov5\\train\\exp6_640_100e\\weights\\best.pt'], imgsz=[640, 640], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=Fa
lse, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['torchscript', 'onnx']
YOLOv5  v6.1-11-g63ddb6f torch 1.9.0 CPU

Fusing layers...
Model Summary: 213 layers, 7015519 parameters, 0 gradients, 15.8 GFLOPs

PyTorch: starting from Z:\yolov5\train\exp6_640_100e\weights\best.pt with output shape (1, 25200, 7) (14.4 MB)

TorchScript: starting export with torch 1.9.0...
TorchScript: export success, saved as Z:\yolov5\train\exp6_640_100e\weights\best.torchscript (28.5 MB)

ONNX: starting export with onnx 1.10.2...

(yolo) C:\Developer\Git\Inspection\AI\yolo\yolov5>**

**Comment by apanand14** [comment]
I have just uninstalled onnx 1.10.2 and installed latest 1.11.0 and it worked.!! Do not what's the problem with onnx -1.10.2

But still, Any of yyou know what was the issue with onnx version 1.10.2 then please let me know. Thank you!!

**(yolo) C:\yolo\yolov5>python export.py --weights Z:\yolov5\train\exp_640_50e\best.pt
export: data=data\coco128.yaml, weights=['Z:\\yolov5\\train\\exp_640_50e\\best.pt'], imgsz=[640, 640], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnost
ic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['torchscript', 'onnx']
YOLOv5  v6.1-11-g63ddb6f torch 1.9.0 CPU

Fusing layers...
Model Summary: 213 layers, 7015519 parameters, 0 gradients, 15.8 GFLOPs

PyTorch: starting from Z:\yolov5\train\exp_640_50e\best.pt with output shape (1, 25200, 7) (14.4 MB)

TorchScript: starting export with torch 1.9.0...
TorchScript: export success, saved as Z:\yolov5\train\exp_640_50e\best.torchscript (28.5 MB)

ONNX: starting export with onnx 1.11.0...
ONNX: export success, saved as Z:\yolov5\train\exp_640_50e\best.onnx (28.5 MB)

Export complete (4.92s)**
